LibraryManagement.Data/DataRequestObjects/Genres/InsertGenres.cs

```diff
@@ -0,0 +1,29 @@
+﻿using LibraryManagement.Data.Abstraction;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace LibraryManagement.Data.DataRequestObjects.Genres
+{
+    public class InsertGenres : IDataExecute
+    {
+        public string _Name { get; set; }
```

Comment:
Since this is a public property it would be best to use PascalCase for the naming.... using _cameCase with an underscore is typically for private fields